Applications/QuickShow/QSWidget.cpp

```diff
@@ -247,6 +253,21 @@ void QSWidget::load_from_file(const String& path)
     relayout();
 }
 
+void QSWidget::switch_main_windows_to_full_screen()
+{
+    auto win = this->window();
+
+    win->set_fullscreen(!win->is_fullscreen());
+
+    for (auto& child : win->main_widget()->children()) {
+        if (child.name() == "toolbar_container") {
+            auto& toolbar = static_cast<GUI::ToolBarContainer&>(child);
+            toolbar.set_visible(!win->is_fullscreen());
+            break;
+        }
+    }
```

Comment:
Instead of doing this in `QSWidget`, let's move it to `main()` which is really the "owner" of the window.
I would suggest something like this:

```cpp
class QSWidget {
    Function<void()> on_doubleclick;
};

void QSWidget::doubleclick_event(GUI::MouseEvent&)
{
    on_doubleclick();
}

int main()
{
    ...
    qswidget->on_doubleclick = [&] {
        window->set_fullscreen(!window->is_fullscreen());
    };
}

```